tiledb/sm/cpp_api/config.h

```diff
@@ -745,6 +745,19 @@ class Config {
     return val;
   }
 
+  /**
+   * Check if a configuration parameter exists.
+   * @param param Name of configuration parameter
+   * @return true if the parameter exists, false otherwise
+   */
+  bool contains(const std::string& param) const {
```

Comment:
We don't have a style guide entry on this yet, but I believe `string_view` is preferred over constant reference `std::string` in C++17.